homeassistant/components/squeezebox/config_flow.py

```diff
@@ -222,3 +242,55 @@ async def async_step_dhcp(
 
         # if the player is unknown, then we likely need to configure its server
         return await self.async_step_user()
+
+
+class OptionsFlowHandler(OptionsFlow):
+    """Options Flow Handler."""
+
+    def __init__(self, config_entry: ConfigEntry) -> None:
+        """Initialize options flow."""
+        self.config_entry = config_entry
+
+    async def async_step_init(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Options Flow Steps."""
+
+        errors: dict[str, str] = {}
+
+        async def _validate_input(data: dict[str, Any]) -> str | None:
+            """Validate the user input allows us to connect."""
+
+            if data[CONF_BROWSE_LIMIT] < 1 or data[CONF_VOLUME_STEP] < 1:
+                raise ValueError
+
+            return None
+
+        if user_input is not None:
+            try:
+                await _validate_input(user_input)
+            except ValueError:
+                errors["base"] = "value"
+            if not errors:
+                return self.async_create_entry(title="", data=user_input)
+
+        return self.async_show_form(
+            step_id="init",
+            data_schema=vol.Schema(
+                {
+                    vol.Optional(
+                        CONF_BROWSE_LIMIT,
+                        default=self.config_entry.options.get(
+                            CONF_BROWSE_LIMIT, DEFAULT_BROWSE_LIMIT
+                        ),
+                    ): int,
+                    vol.Optional(
+                        CONF_VOLUME_STEP,
+                        default=self.config_entry.options.get(
+                            CONF_VOLUME_STEP, DEFAULT_VOLUME_STEP
+                        ),
+                    ): int,
+                }
```

Comment:
Could we use a schema constant, with suggested_values?

Reply:
It could be, but the other schemas are defined in this way, so keeping the approach consistent.

Reply:
I insist, it make it much clearer to distinguish what is part of the schema constant, and what is part the entry data (the suggested_values which get merged in)

Reply:
Also => defaults get added when the user leaves the field empty, whereas suggested values are added during the diplay of the form, and they make it possible to clear fields without the defaults getting injected unexpectedly

Reply:
OK - using the number selectors now.  They work nicely.  I tried using the select() function as per the docs as well as that makes it even easier to read, but that doesn't seem to work with suggested values sadly.